src/shared/constants/round-phase.js

```diff
@@ -0,0 +1,3 @@
+export const COUNTDOWN = 'COUNTDOWN';
+export const IN_PROGRESS = 'IN_PROGRESS';
+export const FINISHED = 'FINISHED';
```

Comment:
Do we consider to have namespaces in the future? I am not very familiar with the whole app, but I can assume that FINISHED can be the phase of the game as well.